src/components/KCatalog/KCatalog.vue

```diff
@@ -608,70 +538,36 @@ onMounted(() => {
 </script>
 
 <style lang="scss" scoped>
-.k-card-catalog {
-  .k-card-catalog-title {
-    color: var(--kui-color-text-neutral-stronger, $kui-color-text-neutral-stronger);
-  }
-
-  .k-catalog-page {
-    display: grid;
-    grid-gap: var(--kui-space-80, $kui-space-80);
-    grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
+.k-catalog {
+  display: flex;
+  flex-direction: column;
+  gap: var(--kui-space-70, $kui-space-70);
+
+  .catalog-title {
+    font-family: var(--kui-font-family-text, $kui-font-family-text);
+    font-size: var(--kui-font-size-60, $kui-font-size-60);
+    font-weight: var(--kui-font-weight-bold, $kui-font-weight-bold);
+    letter-spacing: var(--kui-letter-spacing-minus-40, $kui-letter-spacing-minus-40);
```

Comment:
question: do we want to customize the letter spacing here? If the user decides to utilize a non `h*` tag they may not want to alter the default letter spacing

Reply:
I just used the same styles as KCard so it's consistent with other headings. I also don't mind removing the `letter-spacing` property here if you think that will be better

Reply:
Since we don't control the font family used with Kongponents, I really don't think we should be customizing letter spacing (here, or in KCard, etc.)

Reply:
Makes sense - removed `letter-spacing` from here and other components.